simpletransformers/t5/t5_model.py

```diff
@@ -104,7 +111,10 @@ def __init__(
             self.args.fp16 = False
 
         self.args.model_type = "T5"
-        self.args.model_name = model_name
+        if model_name==None:
+            self.args.model_name = "T5fromScratch"
+        else:
+            self.args.model_name = model_name
```

Comment:
Is this change necessary? `LanguageModelingModel` just leaves the `model_name` blank when training from scratch. I don't _think_ it will cause any issues though.

Reply:
I've tried to leave this blank but it causes errors in saving/loading because the `model_name` is a textual part of the paths and `None` is not allowed.

Reply:
Oh yeah, that makes sense.